docs/content/docs/ops/state/state_backends.md

```diff
@@ -170,7 +170,7 @@ A sample section in the configuration file could look as follows:
 
 ```yaml
 # The backend that will be used to store operator state checkpoints
-state.backend: hashmap
+state.backend.type: hashmap
```

Comment:
I suggest 
- changing the other instances in this file 
- changing the reference to this in savepoints.md as well
- changing the zh versions also.